apps/cowswap-frontend/src/modules/tokensList/pure/TokenInfo/index.tsx

```
@@ -11,6 +16,12 @@ export interface TokenInfoProps {
 
 export function TokenInfo(props: TokenInfoProps) {
   const { token, className } = props
+  const [isCopied, setCopied] = useCopyClipboard()
```

ESLint

[error] 'isCopied' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)